docs/api/odbc.md

```diff
@@ -0,0 +1,116 @@
+Someone can see the ODBC API composed of the Driver Manager (DM) and the ODBC drivers.
+
+The DM is part of the system library, e.g., unixODBC, which manages the communications between the user applications and the ODBC drivers.
+Typically, applications are linked against the DM, which uses Data Source Name (DSN) to look up the correct ODBC driver.
+<!--- with dynamically linkage call the ODBC driver. -->
+
+The ODBC driver is a DBMS implementation of the ODBC API, which handles all the internals of that DBMS.
+
+The DM maps user application calls of ODBC functions to the correct ODBC driver that performs the specified function and returns the proper values.
+
+## DuckDB ODBC Assets
+
+DuckDB supports the ODBC version 3.0 according to the [Core Interface Conformance](https://docs.microsoft.com/en-us/sql/odbc/reference/develop-app/core-interface-conformance?view=sql-server-ver15). 
+
+We release the ODBC driver as assets for Linux and Windows.
+Users can download them from the [Latest Release of DuckBD](https://github.com/duckdb/duckdb/releases).
+
+There, there are two assets: **duckdb\_odbc-linux-amd64.zip** and **duckdb\_odbc-windows-amd64.zip**).
+These assets have specific driver artifacts to the corresponding operating system.
+
+## Linux Setup
+The ODBC setup on Linux is based on files, the well-known **.odbc.ini** and **.odbcinst.ini**.
+These files can be placed at the system `/etc` directory or at the user home directory `/home/<user>` (shortcut as `~/`).
+The DM prioritizes the user configuration files and then the system files.
+
+### The ".odbc.ini" File
+
+The **.odbc.ini** contains the DSNs for the drivers, which can have specific knobs.
+
+An example of **.odbc.ini** with DuckDB would be:
+
+```
+[DuckDB]
+Driver = DuckDB Driver
+Database=:memory:
+```
+
+**[DuckDB]**: between the brackets is a DSN for the DuckDB.
+
+**Driver**: it describes the driver's name, and other configurations will be placed in the **.odbcinst.ini**.
+
+**Database**: it describes the database name used by DuckDB, and it can also be a file path to a `.db` in the system.
+
+### The ".odbcinst.ini" File
+
+The **.odbcinst.ini** contains general configurations for the ODBC installed drivers in the system.
+A driver section starts with the driver name between brackets, and then it follows specific configuration knobs belonging to that driver.
+
+An example of **.odbcinst.ini** with the DuckDB driver would be:
+
+```
+[ODBC]
+Trace = yes
+TraceFile = /tmp/odbctrace
+
+[DuckDB Driver]
+Driver = /home/<user>/duckdb/build/release/tools/odbc/libduckdb_odbc.so
+```
+
+
+**[ODBC]**: it is the DM configuration section.
+
+**Trace**: it enables the ODBC trace file using the option `yes`.
+
+**TraceFile**: the absolute system file path for the ODBC trace file.
+
+
+**[DuckDB Driver]**: the section of the DuckDB installed driver.
+
+**Driver**: the absolute system file path of the DuckDB driver. 
+
+
+
+## Windows Setup
```

Comment:
Perhaps we can split off the ODBC setup into separate files similar to the Snowflake pages, so we have e.g. `ODBC/Linux Setup`, `ODBC/Windows Setup`, etc